### PR TITLE
fix: bump deps

### PR DESCRIPTION
**Pull request by peternhale**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,9 +5,9 @@
   "author": "Salesforce",
   "bugs": "https://github.com/oclif/plugin-plugins/issues",
   "dependencies": {
-    "@oclif/color": "^1.0.0",
+    "@oclif/color": "^0.1.2",
```
Comment (RodEsp) [inline package.json:8]: Why downgrade this version?

File: package.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
   "devDependencies": {
     "@commitlint/config-conventional": "^12.1.4",
     "@oclif/plugin-help": "^5.1.7",
-    "@oclif/test": "^2.0.2",
+    "@oclif/test": "^1.2.8",
```
Comment (RodEsp) [inline package.json:24]: Why downgrade this version?

**Review by RodEsp — COMMENTED** [review]
Are there supposed to be changes to the circleci yml in this PR?
The title seems to suggest it that but there aren't any.

File: package.json (hunk 2 above)
Comment (peternhale) [reply package.json:24]: with 2.0.2 the unit tests are failing. Mike and I discussed and chose to use v1* for the mean time and log a bug against oclif/test to see what was happening.

File: package.json (hunk 1 above)
Comment (peternhale) [reply package.json:8]: Same as below

**Review by shetzel — APPROVED** [review]

**Comment by peternhale** [comment]
> @peternhale Are there supposed to be changes to the circleci yml in this PR? The title seems to suggest it that but there aren't any.

Looks like the change was already made in another PR and my change diffed 0 against main for that file.

**Review by RodEsp — APPROVED** [review]

**Pull request merged**